src/modules/auth/token.service.spec.ts

```diff
@@ -106,18 +106,14 @@ describe('TokenService', () => {
     await app.init();
   });
 
-  it('should have same GET request ORIGIN header as stored in JWT token ', async () => {
+  it('should authorize if token origin matches request origin', async () => {
     return mockRequest('get', 'https://switchboard-dev.energyweb.org');
   });
 
-  it('should NOT have same POST request ORIGIN header as stored in JWT token ', async () => {
+  it('should not authorize if token origin does not matches request origin ', async () => {
     return mockRequest('post', 'https://hacked-website.com', 401);
   });
 
-  it('should NOT have origin in request headers', async () => {
```

Comment:
removed because this test allows empty request origin when origin of access token is set